spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/scutilities/HttpHandler.java

```diff
@@ -64,21 +66,33 @@ private HttpHandler() {
     public Observable<Response> get(final String url) throws IOException {
         return Observable.create(new Observable.OnSubscribe<Response>() {
             @Override
-            public void call(Subscriber<? super Response> subscriber) {
-                try {
-                    Request request = new Request.Builder()
-                            .url(url)
-                            .build();
-                    Response response = client.newCall(request).execute();
-                    subscriber.onNext(response);
-                    subscriber.onCompleted();
-                    if (!response.isSuccessful()) subscriber.onError(new Exception("error"));
-                } catch (IOException e) {
-                    subscriber.onError(e);
-                }
+            public void call(final Subscriber<? super Response> subscriber) {
+                SCBackendService.networkConnected
```

Comment:
Remove this b/c we want to be able to use the HttpHandler without the backend service